content/en/docs/guides/Getting Started/_index.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Better if we don't use spaces in the folder names, that messes up navigation in a terminal :grimacing: 

Let's use snake_case instead.

Reply:
Solved